bigquery/deploy/deploy-indexer.sh

```diff
@@ -23,8 +23,10 @@ echo
 
```

Comment:
I get: `ERROR: (gcloud.container.node-pools.create) ResponseError: code=409, message=Already exists: projects/biobank-explorer/zones/us-central1-a/clusters/elasticsearch-cluster/nodePools/indexer-pool.`

If the node pool already exists, this script should either use it or delete and recreate it. The script shouldn't fail.